public/locales/en-US/translations.json

```diff
@@ -1,7 +1,8 @@
 {
   "testnet_data": "testnet data",
-  "live_data": "live data",
+  "mainnet_data": "live data",
   "devnet_data": "devnet data",
+  "nft_sandbox_data": "NFT sandbox data",
```

Comment:
IMO this should be lowercase to match the other 3

Reply:
Discussed offline, this doesn't work because of the way capitalization happens (it would be capitalized as `Nft`)